keps/sig-scheduling/4816-dra-prioritized-list/kep.yaml

```diff
@@ -27,18 +28,18 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.32"
+latest-milestone: "v1.33"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.32"
-  beta: "v1.33"
-  stable: "v1.34"
+  alpha: "v1.33"
+  beta: "v1.34"
+  stable: "v1.35"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: DRAFirstAvailableOf
+  - name: DRAPrioritizedList
```

Comment:
`DRAFirstAvailable` is mentioned in the README.md.

Both works for me, with a slight preference for `DRAPrioritizedList` because it matches the title of the KEP.

Reply:
The code PR uses `DRAPrioritizedList`, so let's fix the name in the README.md.